qiskit_ibm/managed/managedjob.py

```diff
@@ -120,9 +118,9 @@ def _async_submit(
                         job_tags=job_tags,
                         **run_config)
                 except IBMQBackendJobLimitError:
-                    final_states = [state.value for state in API_JOB_FINAL_STATES]
-                    oldest_running = backend.jobs(limit=1, descending=False,
-                                                  db_filter={"status": {"nin": final_states}})
+                    non_final_job_status = ['INITIALIZING', 'QUEUED', 'VALIDATING', 'RUNNING']
```

Comment:
Has to be one of the non final job statuses from `JobStatus` - otherwise, there will be a `IBMQBackendValueError` raised from `get_status_filter`

Reply:
Hmm in that case can we use JOB_FINAL_STATES
https://github.com/Qiskit/qiskit-terra/blob/b757914b7200b2a4a4e8065477342f25a544df83/qiskit/providers/jobstatus.py#L30
and get the remaining non final states from JobStatus class? So it's easier to maintain?